src/app/shared/comment-editor/markdown-toolbar/markdown-toolbar.component.html

```diff
@@ -2,64 +2,64 @@
 <!-- Provided by a third-party library: https://github.com/github/markdown-toolbar-element -->
 <markdown-toolbar [attr.for]="forTextAreaId">
   <md-bold>
-    <button mat-icon-button color="accent">
+    <button matTooltip="Add bold text, <Ctrl+b>" mat-icon-button color="accent">
```

Comment:
The `<Ctrl+b>` should be removed, since that keyboard shortcut is not valid for Mac